src/main/java/org/broadinstitute/hellbender/metrics/PerUnitInsertSizeMetricsCollector.java

```diff
@@ -162,65 +151,63 @@ public void addMetricsToFile(final MetricsFile<InsertSizeMetrics, Integer> file)
     }
 
     /**
-     * Combine this InsertSizeMetricsCollector with sourceCollector and return a merged InsertSizeMetricsCollector.
-     * @param sourceCollector InsertSizeMetricsCollector to merge in
+     * Combine this InsertSizeMetricsCollector with sourceCollector and return a combined InsertSizeMetricsCollector.
+     * @param sourceCollector InsertSizeMetricsCollector to combine in
      * @return InsertSizeMetricsCollector representing the combination of the source collector with this collector
      */
     public PerUnitInsertSizeMetricsCollector combine(PerUnitInsertSizeMetricsCollector sourceCollector) {
-        String validationMessage = "Internal error combining collectors";
-        validateEquals(this.sample, sourceCollector.sample, validationMessage);
-        validateEquals(this.library, sourceCollector.library, validationMessage);
-        validateEquals(this.readGroup, sourceCollector.readGroup, validationMessage);
+        Utils.nonNull(sourceCollector);
+        Objects.equals(this.sample, sourceCollector.sample);
```

Comment:
hmm, this does not work. Objects.equals just returns false when the args are not equal but this should blow up. Need to wrap in an if statement